Week-6/Assignment-3.js

```diff
@@ -0,0 +1,27 @@
+function sort_array(input_array){
+    console.log(input_array)
```

Comment:
@soumya213 Please try to use the same array and place the items in correct order. dont create multiple arrays